cli/lsp/tsc.rs

```diff
@@ -3897,13 +3897,12 @@ fn op_load<'s>(
 }
 
 #[op2]
-fn op_resolve<'s>(
-  scope: &'s mut v8::HandleScope,
+#[serde]
+fn op_resolve(
   state: &mut OpState,
   #[serde] args: ResolveArgs,
-) -> Result<v8::Local<'s, v8::Value>, AnyError> {
+) -> Result<Vec<Option<(String, String)>>, AnyError> {
   let state = state.borrow_mut::<State>();
-  let mark = state.performance.mark_with_args("tsc.op.op_resolve", &args);
```

Comment:
I had to remove this mark. In the fresh repo, moving the cursor across the screen once adds many thousands of lines because of this.

Reply:
Isn't that an indicator that this ops is called too many times? Maybe we could cache this info in TSC somehow?